internal/service/finspace/kx_cluster_test.go

```diff
@@ -1007,6 +1235,167 @@ resource "aws_finspace_kx_cluster" "test" {
 `, rName))
 }
 
+func testAccKxClusterConfig_rdbEnvironmentId(rName, environmentId string) string {
+	return acctest.ConfigCompose(
+		testAccKxClusterConfigBase(rName),
+		fmt.Sprintf(`
+resource "aws_finspace_kx_cluster" "test" {
+  name                 = %[1]q
+  environment_id       = %[2]q
+  type                 = "RDB"
+  release_label        = "1.0"
+  az_mode              = "SINGLE"
+  availability_zone_id = aws_finspace_kx_environment.test.availability_zones[0]
+
+  savedown_storage_configuration {
+    type = "SDS01"
+    size = 500
+  }
+
+  capacity_configuration {
+    node_count = 2
+    node_type  = "kx.s.xlarge"
+  }
+
+  vpc_configuration {
+    vpc_id             = aws_vpc.test.id
+    security_group_ids = [aws_security_group.test.id]
+    subnet_ids         = [aws_subnet.test.id]
+    ip_address_type    = "IP_V4"
+  }
+}
+`, rName, environmentId))
+}
+
+func testAccKxClusterConfig_rdbDatabaseName(rName, databaseName string) string {
```

Comment:
should we name this invalidDatabaseName or something similar? databaseName field seems like it should be assigned to name field of database otherwise, but that's not what we're doing (since we're testing invalid case)

Reply:
in that case, would it make sense to hard code an invalid value to the template and remove the `databaseName` parameter?